### DISCUSSION TITLE
Use Media Query in ::slotted()

### DISCUSSION THREAD

**Discussion by mojtaba-afraz** [lit/lit#3167] [Q&A] [answered]
Hello Guys ✌️
I want to use media query in ::slotted() but i getting error 💀; anybody can Help me !?

**Comment by sorvell** [comment, marked as answer]
Here's an [example](https://lit.dev/playground/#gist=cb9960a055b8555ed9446d80716d2694).

Resize the preview pane to be narrower/wider than 400px to see the media query take affect.

**Reply by mojtaba-afraz** [reply]
thank you Steve ✌️
